05_sumAll/sumAll.js

```diff
@@ -1,6 +1,22 @@
-const sumAll = function() {
-
+const sumAll = function (firstArg, secArg) {
+  let sum = 0;
+  if (
+    typeof firstArg != "number" ||
+    typeof secArg != "number" ||
+    firstArg <= 0 ||
+    secArg <= 0
+  ) {
+    return "ERROR";
+  }
+  if (firstArg > secArg) {
+    const temp = firstArg;
+    firstArg = secArg;
+    secArg = temp;
```

Comment:
You are modifying original arguments
<img src="https://media0.giphy.com/media/L7zmmuaEo50MCt1Y7o/giphy.gif?cid=5a38a5a2hwt8a0chuzpgy10dpty7cayk8umkl1q30wxszeqw&rid=giphy.gif&ct=g" />

Reply:
You could read about `Math.min`, and `Math.max`, should also help.